options/locale/locale_en-US.ini

```diff
@@ -1305,7 +1305,8 @@ issues.filter_label_no_select = All labels
 issues.filter_milestone = Milestone
 issues.filter_milestone_no_select = All milestones
 issues.filter_project = Project
```

Comment:
Why do we have `issues.filter_project`, and `issues.filter_projects`, and both mean completely different things?
One of the two should be renamed…

Reply:
Looks like not only `project`, also others `filter_label` and `filter_labels`. So I think this should be leaved to other PR.